src/pages/Book/BookARsv.js

```diff
@@ -0,0 +1,263 @@
+import React from 'react';
+import styled from 'styled-components';
+import Calendar from 'react-calendar';
+import 'react-calendar/dist/Calendar.css';
+import { useState } from 'react';
+
+const BookARsv = () => {
+  const [value, setValue] = useState(new Date());
+
+  const changeDate = e => {
+    setValue(e);
+  };
+
+  return (
+    <>
+      <ReservationContainer>
+        <PlaceSelect>
+          <Step1>STEP1</Step1>
+          <Place>지역/영화관 선택</Place>
+          <PlacePick>
+            <PlacePickTextSeoul>
+              <PlaceTextBox>
+                <PlacePickP>서울</PlacePickP>
+              </PlaceTextBox>
+              <PlacePickButtonContainer>
+                <PlaceButton>강남점</PlaceButton>
+                <PlaceButton>선릉점</PlaceButton>
+              </PlacePickButtonContainer>
+            </PlacePickTextSeoul>
+          </PlacePick>
+        </PlaceSelect>
+        <CalendarContainer>
+          <PlaceText>
+            <Step2>STEP2</Step2>
+            <SelectDate>날짜 선택</SelectDate>
+          </PlaceText>
+          <Calendar className="calendar" onChange={changeDate} value={value} />
+        </CalendarContainer>
+        <SelectTime>
+          <Step3>STEP3</Step3>
+          <Time>시간 선택</Time>
+          <TimePickContainer>
+            <TimePick>
+              <TimeButton>🌞 10:10 AM</TimeButton>
+              <TimeButton>🌕 19:45 PM</TimeButton>
+            </TimePick>
+          </TimePickContainer>
+        </SelectTime>
+      </ReservationContainer>
+      <ButtonBox>
+        <SeatButton>좌석 선택</SeatButton>
+      </ButtonBox>
+    </>
+  );
+};
+
+const ReservationContainer = styled.section`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 1200px;
+  height: 350px;
+  margin-top: 60px;
+  border: 1px solid #ecedf2;
+  border-radius: 10px;
+  -webkit-box-shadow: 7px 7px 21px -8px rgba(0, 0, 0, 0.42);
+  box-shadow: 7px 7px 21px -8px rgba(0, 0, 0, 0.42);
+  overflow: hidden;
+`;
+
+const PlaceSelect = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 350px;
+  height: inherit;
+  border-right: 1px solid #ecedf2;
+  padding: 30px 50px;
+  overflow: scroll;
+`;
+
+const Step1 = styled.div`
```

Comment:
```suggestion
const StepTitle = styled.div`
```

위의 리뷰를 반영하면 여기 이 스타일 하나에서 관리가 가능해지겠습니다!